data/processing/input_latency.html

```diff
@@ -155,20 +155,39 @@
     const associatedEvents = latency.associatedEvents;
 
     for (const event of associatedEvents) {
-      if (event.hasOwnProperty('parentContainer') && event.parentContainer.name === 'CrRendererMain')
+      if (event.title === 'RenderWidgetInputHandler::OnHandleInputEvent' &&
+	  event.args.hasOwnProperty('event') &&
+	  event.args.event === LATENCY_EVENT)	  
         return event.parentContainer.getProcess().pid;
     }
   }
 
-  // Get all renderer events given an id 
-  function getRendererEvents(browserHelper, rendererID) {
-    return browserHelper.getAllAsyncSlicesMatching(function(slice) {
-      return (slice.parentContainer.getProcess().pid === rendererID &&
-              slice.parentContainer.name === 'CrRendererMain' && 
-              slice.category != 'disabled-by-default-renderer.scheduler' &&
-              slice.category != 'renderer.scheduler' &&
-              slice.category != 'blink.animations,devtools.timeline,benchmark,rail')
-    });
+  // Get the name of the main renderer thread that handled the input of the latency
+  function getThreadByLatency(latency) {
+    const associatedEvents = latency.associatedEvents;
+
+    for (const event of associatedEvents) {
+      if (event.title === 'RenderWidgetInputHandler::OnHandleInputEvent' &&
+	  event.args.hasOwnProperty('event') &&
+	  event.args.event === LATENCY_EVENT)	  
+        return event.parentContainer;
+    }
```

Comment:
Another 'if that happens then something has gone horribly wrong' scenario. I guess in that case it would be like the trace doesn't know which thread ran this event.